src/data/polynomial/induction.lean

```diff
@@ -66,45 +59,23 @@ and it holds for monomials.
 polynomial.induction_on p (h_monomial 0) h_add
 (λ n a h, by { rw ← monomial_eq_C_mul_X at ⊢, exact h_monomial _ _ })
 
-section coeff
-
-theorem coeff_mul_monomial (p : R[X]) (n d : ℕ) (r : R) :
-  coeff (p * monomial n r) (d + n) = coeff p d * r :=
-by rw [monomial_eq_C_mul_X, ←X_pow_mul, ←mul_assoc, coeff_mul_C, coeff_mul_X_pow]
-
-theorem coeff_monomial_mul (p : R[X]) (n d : ℕ) (r : R) :
-  coeff (monomial n r * p) (d + n) = r * coeff p d :=
-by rw [monomial_eq_C_mul_X, mul_assoc, coeff_C_mul, X_pow_mul, coeff_mul_X_pow]
-
--- This can already be proved by `simp`.
-theorem coeff_mul_monomial_zero (p : R[X]) (d : ℕ) (r : R) :
-  coeff (p * monomial 0 r) d = coeff p d * r :=
-coeff_mul_monomial p 0 d r
-
--- This can already be proved by `simp`.
-theorem coeff_monomial_zero_mul (p : R[X]) (d : ℕ) (r : R) :
-  coeff (monomial 0 r * p) d = r * coeff p d :=
-coeff_monomial_mul p 0 d r
-
-end coeff
 
 open submodule polynomial set
-variables {f : R[X]} {I : submodule R[X] R[X]}
+variables {f : R[X]} {I : ideal R[X]}
 
+-- TODO: fix whatever this typo from #5693 was supposed to say
 /--  If the coefficients of a polynomial belong to n ideal contains the submodule span of the
 coefficients of a polynomial. -/
```

Comment:
@adomani, any idea what you meant in #5693?

Reply:
```suggestion
-- TODO: fix whatever this typo from #5693 was supposed to say
/--  If the coefficients of a polynomial belong to an ideal, then that ideal contains
the submodule spanned of the coefficients of the polynomial. -/
```
is what I probably meant to say.